package.json

```diff
@@ -11,11 +11,16 @@
     "url": "https://github.com/vercel/style-guide.git"
   },
   "license": "MPL-2.0",
+  "exports": {
```

Comment:
Did you forget to add `type: module`?

Reply:
I didn't know that was required for `exports` map to work. <picture data-single-emoji=":til:" title=":til:"><img class="emoji" src="https://emoji.slack-edge.com/T0CAQ00TU/til/d5952483dda1dda3.png" alt=":til:" width="20" height="auto" align="absmiddle"></picture>

Reply:
Well with CJS, you don't need to declare any exports - all paths are exported automatically

Reply:
Well yes, but then you have to use exact export path. this mapping lets you have a bit better DX so it can be specified like `@vercel/style-guide/typescript/node` instead of `@vercel/style-guide/typescript/tsconfig.node.json`